metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -183,3 +195,45 @@ def get_report(self):
     def close(self):
         if self.consumer:
             self.consumer.close()
+
+@dataclass
+class TopicSchema:
+    key_schema: Optional[Schema]
+    value_schema: Optional[Schema]
+
+
+class ConfluentSchemaRegistry():
```

Comment:
Might be good to create an abstract base class to make it clear what are the interfaces you expect the "plugin" class to implement.